File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printHashes(int nbr)
{
    for (int i = 0; i < nbr; i++)
    {
        for (int space = 1; space < nbr - i; space++)
            printf(" ");
        for (int left = 0; left <= i; left++)
            printf("#");
        printf("  ");
        for (int right = 0; right <= i; right++)
            printf("#");
        printf("\n");
    }
}

int main(void)
{
    int height;
    do
        height = get_int("Height: ");
    while (height <= 0 || height > 8);
    printHashes(height);
}

File: sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
for i in range(height):
    i += 1
    for space in range(height - i):
        space += 1
        print(" ", end="")
    for left in range(i):
        print("#", end="")
    print("  ", end="")
    for right in range(i):
        print("#", end="")
    print()
